### ISSUE TITLE
Gravity issue: possible breakthrough. Sorry Tsuna, I accidently deleted the email alerts and can't find the original thread

### ISSUE THREAD

**Issue by commche** [tsunamayo/Starship-EVO#4310]
Interesting findings!

TLDR; gravity unit only works if you are a certain distance in space from a ring. Tests showed that warping to a new ring slightly changes the min/max proximity from the ring that gravity works at or not.

I think you may have asked if I built those ships. Yes
and can I boot up test 1 save and gravity doesn't work. yes, still not working on that save.
booted up test save 2. its working, but!!! save 2 is near a station. So I went to the ring, went a little bit into space, built another platform....not working. 

![image](https://user-images.githubusercontent.com/92918376/144357881-e30f5eac-c742-466a-a8fd-a9964a23799a.png)
gravity not working from this proximity to the ring

![image](https://user-images.githubusercontent.com/92918376/144357752-01d758ab-59b9-42b8-9cb5-8268b8a7d64f.png)
From this distance from the ring. its working no probs. You can see the jetpack icon shows that jetpack is off.

![image](https://user-images.githubusercontent.com/92918376/144359639-456c5f72-1a9c-4295-b86d-a5e1014a3771.png)
This distance was about as close as I could get before gravity started failing.

Okay! Tested this theory in both saves. Gravity works if far enough from the ring!
![image](https://user-images.githubusercontent.com/92918376/144360727-a28dfdfa-27ff-4dd4-91d0-4897623919ee.png)
this is save 1. gravity works if far enough from the ring

![image](https://user-images.githubusercontent.com/92918376/144361853-916c3e8a-91f7-405b-a9e4-d9af91b697d7.png)

Warped to another system. A little different here. As you can see Im close to the ring

![image](https://user-images.githubusercontent.com/92918376/144362274-c5c10340-b9ad-4298-b74d-98b212948169.png)
even closer this time. gravity works

![image](https://user-images.githubusercontent.com/92918376/144362388-32ec1382-07f0-449a-a0cb-2c1bc0dc4363.png)
At this proximity, inside but high up. Gravity not working.

**Comment by tsunamayo** [comment]
@commche Thanks a lot for the finding, but I should have fixed the bug already! (that is why you cant find the thread ^^)
Cheers

**Comment by commche** [comment]
oh!!!! damn! Well, nice work! hehe sorry about the spam!
